aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -289,6 +299,11 @@ def create_exporter(self):
         ]:
             temporality_dict[typ] = AggregationTemporality.DELTA
 
+        if _is_lambda_environment():
+            # When running in Lambda, export Application Signals metrics over UDP
+            application_signals_endpoint = os.environ.get(AWS_XRAY_DAEMON_ADDRESS_CONFIG, "127.0.0.1:2000")
```

Comment:
No need hardcode "127.0.0.1:200" since udp exporter has DEFAULT_ENDPOINT

Reply:
Addressed comment offline. This is just a fallback.